### DISCUSSION TITLE
include many new espeak variants in attachment

### DISCUSSION THREAD

**Discussion by joshknnd1982** [nvaccess/nvda#14629] [Ideas]
Please include all the new espeak ng klatt6 speech player variants included in the attached zip file in the new NVDA. There are new baby variants, child, male, female, alien and bug and robot variants, many based on klatt6 nv speech player. With all these new speech player based klatt6 variants, and the improvements in espeak ng the need for eloquence is going away.
[exclamation-v.zip](https://github.com/nvaccess/nvda/files/10700466/exclamation-v.zip)

**Comment by CyrilleB79** [comment]
@josephsl, I am not against closing. But for future trackability, could you provide a reason why you have closed this issu?

**Comment by seanbudd** [comment]
As the reasoning for this feature request is missing I've converted this to a discussion.
It would be good to gauge why this is important to include. For example, do these voices improve the ability to use NVDA, e.g to understand speech, or are they more of a novelty?

**Comment by joshknnd1982** [comment]
some of the variants are a novelty, while most others greatly improve people's ability to both understand speech, use NVDA, and use android and IOS as a result of the new variants.

**Comment by joshknnd1982** [comment]
I have a friend who used eloquence for decades. When I showed him how good espeak has become because of the new variants, particularly the klatt 6 speech player variants, he has since switched and is using espeak insteaf of using eloquence. The new variants really help with how pleasant and understandable espeak ng now is.

**Reply by seanbudd** [reply]
Could you explain which variants are important to include?
Can you explain what the source of this zip file is?
It's not immediately clear where these voices are from. Are they part of the main espeak project https://github.com/espeak-ng/espeak-ng?

**Comment by joshknnd1982** [comment]
yes. The variants which are important to include are: 
adam
adam2
alian
alien
antonio
antonio2
AnxiousAndy2
aunty2
baby
belinda2
benjamin
benjamin2
benjamin3
betty
betty4
betty5
Bill
BNS
bob
bobby
bobby2
bobby3
brian
bug
bugCrushed
caleb2
calebf
caveman2
chad
chad2
chad3
child
chipmunk
chris
croak2
daniel
daniel2
dave
dave2
dave3
dave4
dave5
dave6
david
david2
deepmale
DELTA
Demonic
Demonic2
Denis
Dennis4
Denture
diego
doctor dennis
doctor dennis2
doctor dennis3
doctor dennis4
doctor dennis5
domingo
ed
ed2
eddy
edith
edward
edward2
edward3
elif
evil
f8
f9
fatih
federico
filelist.txt
franco
frank
frank2
gary
giantmonster
giantrobot
givingInstructions
givingInstructions2
glen
grandma
grandpa
harry
harry2
harry3
harry4
HarryRobot
iven
Jacky
Jason
Jason2
josh
josh2
julie2
kit
kit2
kit3
kit4
kit5
Larry2
leann
linda
m9
Mamie
Marco
Mario
marvin
max
max4
mike2
muhammed2
nancy
olivia
osman2
paul2
paul3
Paul3young
quincy
quincy2
read
recep2
Reed
Reed2
RicishayMax3
rob2
robert
robert2
robert3
robosoft8
Rodger
Rodger2
rodrigo
sam
sandy2
sarah
shelby2
shelley
Simon
sonya
sonya2
steph
steph4
Storm
Susan6
tom2
tracy
Tweaky
Uni
ursula2
ursula3
voiceoverartist
wade
wave
wave2
wave3
weirdvariant
whisper
whisperf
yelling
yelling2
zac
ZBondDenture
zira ...
[espeaknewvariants.zip](https://github.com/nvaccess/nvda/files/10718563/espeaknewvariants.zip)

**Comment by joshknnd1982** [comment]
The source of the zip file is from Guillermo from Spain and one or two other people from Chile. The voices are not part of the main espeak project, but it would greatly improve espeak if they were a part of the main espeak project because the Klatt-based-voices or speech-player-based-voices, klatt6 voices, are far more pleasant to listen to than the default espeak voices. I did let a few default non-klatt voices in there for those who like them, and the none variant should still be there because its a default voice variant. But many people including myself really like klatt6 voices. And the file I sent has a lot of them. I also lower espeak's inflection down to 20 or 30 by default because then it sounds more monotone and predictable letting me and others used to eloquence, focus more on what I and we, are reading. For me and others used to eloquence, a very high inflection is distracting when trying to read and edit text. But very high inflections like an inflection of 90 to 100% is good for reading certain kinds of text, so keeping the ability to change inflection or pitch variation is important.

**Reply by seanbudd** [reply]
Without understanding the licensing, we cannot integrate these into NVDA. I would suggest that the creators of these voices

1. Create a PR to the eSpeak project https://github.com/espeak-ng/espeak-ng to make them available to everyone
1. Or, create an NVDA addon which packages these voices.

**Comment by joshknnd1982** [comment]
also for me, and I would guess for others used to eloquence, we find the faster the speech, the lower the inflection. Reading an entertaining short story at a slow speech rate with extremely high pitch variation or inflection is good. Trying to fill out a big online form, using very fast speech, is only good with a very low pitch variation, low inflection of about 20 or 30 percent.

**Comment by Brian1Gaff** [comment]
I think that only the few he mentions are really of any use, However for 
those with long memories, the add on that was created for the old Espeak 
that sounded a bit more like Eloquence was perhaps better than these. Its a 
pity it could not have been made to work on newer code.
 The others seem to be merely tweaks of the ones that exist already. I guess 
to be fair, some of the non silly ones could make certain languages better, 
but  I would not dare to comment.
 I mean given a few hours anyone can tweak a voice on Espeak, as it is well 
documented on  which of the parameters do what.
 However it is still Espeak, based on the same samples, so will always have 
a similarity. One has to beware when creating them not to set the volume of 
certain frequencies too high, as some of the new ones do, as they tend to 
rattle or sound distorted on small laptop speakers. That is why the Quincy 
and Auntie voices I originally made are slightly lower in volume.
 I do find the bare bones None voice particularly annoying as it has a huge 
peak in the presence range, but then each to their own.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by joshknnd1982** [comment]
don't discount the novelty variants. and the variants that pause after every word like the weird variant, child and female klatt variants, even the baby variant. That is because for people with hearing issues and other disabilities, those variants may work great for them.

**Comment by Adriani90** [comment]
see also discussions in issue #561.
